e2e/pod.go

```diff
@@ -276,6 +284,19 @@ func waitForPodInRunningState(name, ns string, c kubernetes.Interface, t int) er
 			return true, nil
 		case v1.PodFailed, v1.PodSucceeded:
 			return false, conditions.ErrPodCompleted
+		case v1.PodPending:
+			if expectedError != "" {
+				events, err := c.CoreV1().Events(ns).List(context.TODO(), metav1.ListOptions{
+					FieldSelector:fmt.Sprintf("involvedObject.name=%s",name)
```

Comment:
I think this hasn't passed `go fmt`?
The line also needs to end in a `,`:
```
level=error msg="Running error: buildir: analysis skipped: errors in package: [/go/src/github.com/ceph/ceph-csi/e2e/pod.go:290:62: missing ',' before newline in composite literal]"
```

Reply:
fixed, thanks !